src/Microsoft.TestPlatform.ObjectModel/DataCollector/DataCollectionContext.cs

```diff
@@ -85,14 +85,14 @@ protected internal DataCollectionContext(SessionId sessionId, TestCase testCase)
     /// Identifies the session under which the data collection occurs.  Will not be null.
     /// </summary>
     [DataMember]
```

Comment:
REVIEW ME: Changes on this type are related to some `DataMember` so there might have been some impact on serialization although this doesn't seem to be part of any old serialization path (XmlSerializer).